### DISCUSSION TITLE
Amazing plugin! Will the .obsidian will ever be native?

### DISCUSSION THREAD

**Discussion by mrmetrix4** [remotely-save/remotely-save#126] [Q&A] [answered]
I'm using the plugin to sync my PC and Android phone, and I'd like them to have the same plugins and themes.
Will the .obsidian ever be fully supported?
Thanks for the great job!

**Comment by fyears** [comment, marked as answer]
Currently it’s supported but deletions inside that folder cannot be synced. You can backup your config and give it try in Advanced settings -> Sync Config Dir
